tests/density_estimator_test.py

```diff
@@ -318,7 +325,9 @@ def test_correctness_of_batched_vs_seperate_sample_and_log_prob(
         input_sample_dim,
     )
     # Batched vs separate sampling
-    samples = density_estimator.sample((1000,), condition=condition)
+    samples = density_estimator.sample(sample_shape, condition=condition)
+    samples = samples.reshape(-1, batch_dim, *input_event_shape)  # Flat for comp.
+
     samples_separate1 = density_estimator.sample(
```

Comment:
nice! I would suggest in a future issue/PR to test this behavior for all batched sample/batched log_prob tests, as in some of them we currently only test the shape.